content/en/modules/nibabel/NiBabel.py

```diff
@@ -0,0 +1,937 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.14.5
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown] slideshow={"slide_type": "slide"}
+# # NiBabel
+#
+# <div style="float: right"><img src="https://nipy.org/nibabel/_static/reggie.png"></div>
+#
+# ## Neuroimaging data and file structures in Python
+#
+# ###### Christopher J Markiewicz
+#
+# ###### NeuroHackademy 2020
+
+# %% [markdown] slideshow={"slide_type": "notes"}
+# The goal of this presentation is to familiarize you with some broad classes of neuroimaging data that can be interacted with in Python, using the NiBabel library.
+#
+# This document is intended to be viewed as a [RISE](https://rise.readthedocs.io/) presentation. It works fine as a notebook, but blocks with images may look strange because they are formatted to work as slides.
+
+# %% [markdown] slideshow={"slide_type": "slide"}
+# # NiBabel
+#
+# NiBabel is a low-level Python library that gives access to a variety of imaging formats, with a particular focus on providing a common interface to the various volumetric formats produced by scanners and used in common neuroimaging toolkits:
+#
+# | | | |
+# |:---: |:---: |:---:|
+# | NIfTI-1 | NIfTI-2 | MGH |
+# | MINC 1.0 | MINC 2.0 | AFNI BRIK/HEAD |
+# | ANALYZE | SPM99 ANALYZE | SPM2 ANALYZE |
+# | DICOM | PAR/REC | ECAT |
+#
+# It also supports surface file formats:
+#
+# | | |
+# |:--:|:--:|
+# | GIFTI | FreeSurfer (FS) geometry |
+# |FS labels | FS annotations |
+#
+# Tractography files:
+#
+# | | |
+# |:--:|:--:|
+# | TrackVis (TRK) | MRtrix (TCK) |
+#
+# As well as the CIFTI-2 format for composite volume/surface data.
+
+# %% [markdown] slideshow={"slide_type": "slide"}
+# ## Installation
+#
+# NiBabel is available on [PyPI](https://pypi.org/project/nibabel/):
+#
+# ```Shell
+# pip install nibabel
+# ```
+#
+# And [conda-forge](https://anaconda.org/conda-forge/nibabel):
+#
+# ```Shell
+# conda install -c conda-forge nibabel
+# ```
+#
+# *Note*: This notebook assumes NiBabel 3+, which requires a minimum Python version of 3.5.
+
+# %% slideshow={"slide_type": "subslide"}
+import nibabel as nb
+print(nb.__version__)
+
+# %% slideshow={"slide_type": "fragment"}
+# Some additional, useful imports
+from pathlib import Path              # Combine path elements with /
+from pprint import pprint             # Pretty-printing
+
+import numpy as np                    # Numeric Python
+from matplotlib import pyplot as plt  # Matlab-ish plotting commands
+from nilearn import plotting as nlp   # Nice neuroimage plotting
+import transforms3d                   # Work with affine algebra
+from scipy import ndimage as ndi      # Operate on N-dimensional images
+# import nibabel.testing                # For fetching test data
```

Comment:
I had to uncomment this line for 
```
tck = nb.streamlines.load(Path(nb.testing.data_path) / 'standard.tck')
trk = nb.streamlines.load(Path(nb.testing.data_path) / 'complex.trk')
```
to work. is that correct?

Reply:
Thanks for catching this. 

I commented this line as pytest was not part of the `requiements.txt` and L90 will cause things to fail. It is now in the PR so we can safely uncommnet that line.
I don't remember if I have run the full notebook in a fresh environment without importing `nibabel.testing`. Testing that now